src/app/shared/pipes/access-role.pipe.spec.ts

```diff
@@ -0,0 +1,49 @@
+/* @format */
+import { AccessRolePipe } from './access-role.pipe';
+
+describe('GetAltTextPipe', () => {
+  let pipe: AccessRolePipe;
+
+  beforeEach(() => {
+    pipe = new AccessRolePipe();
+  });
+  it('create an instance', () => {
+    expect(pipe).toBeTruthy();
+  });
+
+  it('returns the correct access role', () => {
+    const value = 'access.role.viewer';
+    const result = pipe.transform(value);
+    expect(result).toEqual('Viewer');
+  });
+
+  it('returns the correct access role', () => {
```

Comment:
Though these are really basic tests that shouldn't ever fail, I think we should give each test a unique name so that when one fails it communicates which access role isn't being converted properly.